units/systemd-firstboot.service

```diff
@@ -15,7 +15,7 @@ ConditionPathIsReadWrite=/etc
 ConditionFirstBoot=yes
 
 DefaultDependencies=no
-After=systemd-remount-fs.service systemd-sysusers.service systemd-tmpfiles-setup.service
+After=systemd-remount-fs.service systemd-tmpfiles-setup.service
```

Comment:
hmm, is it really true that they can run in any order?

i.e. let's say firstboot gets a headstart, determines that root pw is not set yet, queries the user. now sysusers gets to run, also sees root pw is not set yet. pulls in the root pw from passed system credentials. now user responds and overrides what was imported. bad.

or do we keep the file lock on the passwd db the whole time before we determine the root pwd isn't set and we query the user inetractively? (too lazy to check right now)

Reply:
You are right, it's better to keep this. I really think it was added by mistake, but it seems it was a good mistake. I'll drop this patch.